requirements/frozen.py2.txt

```diff
@@ -323,7 +323,7 @@ Werkzeug==0.14.1
 widgetastic.core==0.32.1
 widgetastic.patternfly==0.0.41
 widgetsnbextension==3.4.2
-wrapanapi==3.0.43
+git+git://github.com/digitronik/wrapanapi.git@hot-add
```

Comment:
:+1:

Reply:
https://pypi.org/project/wrapanapi/

3.0.44 includes the hot-swap properties needed for this test, please update all relevant requirements files.